packages/p2p-v2/src/components/PaymentMethodsFormFooter/PaymentMethodsFormFooter.tsx

```diff
@@ -21,33 +21,39 @@ type TPaymentMethodsFormFooterProps = {
  * @returns {JSX.Element}
  * @example <PaymentMethodsFormFooter actionType={actionType} handleGoBack={handleGoBack} isDirty={isDirty} isSubmitting={isSubmitting} isValid={isValid} />
  * **/
-
 const PaymentMethodsFormFooter = ({
     actionType,
     handleGoBack,
     isDirty,
     isSubmitting,
     isValid,
-}: TPaymentMethodsFormFooterProps) => (
-    <div className='p2p-v2-payment-method-form__buttons' role='payment-methods-form-footer'>
-        <Button
-            className='p2p-v2-payment-method-form__buttons--cancel'
-            onClick={e => {
-                e.preventDefault();
-                e.stopPropagation();
+}: TPaymentMethodsFormFooterProps) => {
+    const { isMobile } = useDevice();
+    const textSize = isMobile ? 'lg' : 'sm';
+
+    return (
+        <div className='p2p-v2-payment-methods-form-footer' role='payment-methods-form-footer'>
+            <Button
+                className='border-2'
+                color='black'
+                onClick={e => {
+                    e.preventDefault();
+                    e.stopPropagation();
```

Comment:
are these needed?

Reply:
yes we need to leave this or else this will cause a glitch when you try to Cancel the edit/add payment methods, it will create/save it automatically